deny.toml

```diff
@@ -1,15 +1,24 @@
+[graph]
+targets = []
 all-features = false
 no-default-features = false
 
+[output]
+feature-depth = 1
+
 [advisories]
-vulnerability = "deny"
```

Comment:
Any reasons to remove those settings?

Reply:
Removing that we hope that if default will be changed, we will notice.
I don't have that trust.

Reply:
I see, I missed that. I guess this is fine then.